<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XUAlarmClock</class>
 <widget class="QWidget" name="XUAlarmClock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="AlarmclockreturnButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="AlarmclockNewButton">
   <property name="geometry">
    <rect>
     <x>944</x>
     <y>0</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListWidget" name="AlarmclockalarmlistWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>1024</width>
     <height>480</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="AlarmclockNewWidget" native="true">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>630</width>
     <height>70</height>
    </rect>
   </property>
   <widget class="QPushButton" name="AlarmclockCancelButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AlarmclockConfirmButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>0</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="AlarmclocktimeEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>0</y>
      <width>350</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="AlarmclockEdithourButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>时</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AlarmclockEditminuteButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>0</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>分</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="AlarmclockDeleteButton">
   <property name="geometry">
    <rect>
     <x>864</x>
     <y>0</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>AlarmclockNewWidget</zorder>
  <zorder>AlarmclockreturnButton</zorder>
  <zorder>AlarmclockNewButton</zorder>
  <zorder>AlarmclockalarmlistWidget</zorder>
  <zorder>AlarmclockDeleteButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
